lib/elixir_search_extractor/search_keyword/search_keywords.ex

```diff
@@ -0,0 +1,56 @@
+defmodule ElixirSearchExtractor.SearchKeyword.SearchKeywords do
+  alias ElixirSearchExtractor.ElixirSearchExtractorWorker.KeywordSearchWorker
+  alias ElixirSearchExtractor.Repo
+  alias ElixirSearchExtractor.SearchKeyword.Schemas.Keyword
+  alias ElixirSearchExtractor.SearchKeywords.Errors.KeywordNotCreatedError
+  alias ElixirSearchExtractor.SearchKeywords.Errors.KeywordNotUpdatedError
+
+  def store_keywords(keyword_list, keyword_file_id) do
+    Enum.each(keyword_list, fn keyword ->
+      create_keyword(%{
+        title: keyword,
+        keyword_file_id: keyword_file_id
+      })
+    end)
+
+    :ok
+  end
+
+  def update_keyword(%Keyword{} = keyword, attrs) do
+    case keyword
+         |> Keyword.changeset(attrs)
+         |> Repo.update() do
```

Comment:
I think it easier to read like this
```suggestion
     keyword
     |> Keyword.changeset(attrs)
     |> Repo.update()
     |> case do
```